src/Chapter2.hs

```diff
@@ -630,6 +630,13 @@ takeEven [] = []
 takeEven [x] = [x]
 takeEven (x : _ : xs) = x : takeEven xs
 
+takeEvenGo :: [Int] -> [Int]
+takeEvenGo xs = go xs 
+  where 
+    go [] = []
+    go (x:xs) 
+      | x `mod` 2 == 1 = x : go xs
+      | otherwise = go xs
```

Comment:
It looks like this solution with `mod` takes even numbers themselves, not numbers on even positions. If you want to implement this function using `go` and checking the position, you need to track the position itself somehow.

Nevertheless, you already solved the task! :partying_face:

Reply:
nice!  `acc` counter used  :laughing: